pandas/tests/extension/test_numpy.py

```diff
@@ -27,7 +28,10 @@
 import pandas as pd
 import pandas._testing as tm
 from pandas.core.arrays.numpy_ import PandasArray
-from pandas.core.internals import managers
+from pandas.core.internals import (
+    blocks,
+    managers,
+)
 from pandas.tests.extension import base
 
 # TODO(ArrayManager) PandasArray
```

Comment:
can you add a note here that these are for mocking (i know seems obvious, but a future reader may not be so clear)

Reply:
sure, ok for follow-up?  (CI is a few minutes from green)